### PR TITLE
Update 08-simulations.md

### PR DESCRIPTION
**Pull request by fairyeszhuo**
modified several typos

### CODE REVIEW

File: _episodes/08-simulations.md [hunk 1]
```diff
@@ -29,7 +29,7 @@ keypoints:
 > 
 > The PhD students have to think about the **numerical implementation** of their solution in the model.
 > 
-> PhD students will also work in pairs and will use the **same model version** (CESM1.2.2 with CAM 5.3) but can set-up a more appropriate compset if needed.
+> PhD students will also work in pairs and will use the **same model version** (CESM1.2.2 with CAM 5.3) but can set up a more appropriate compensate if needed.
```
Comment (annefou) [inline _episodes/08-simulations.md]: No here we really mean compset. compset means "Component set definitions".

See http://www.cesm.ucar.edu/models/cesm2/config/compsets.html

Comment (annefou) [inline _episodes/08-simulations.md]: Here we also need to update it. We are using CESM 2.1.0 with CAM 6 (and not CESM 1.2.2 with CAM 5,3).

Comment (hermanfugl) [inline _episodes/08-simulations.md]: Heads up, _compset_ is the correct word here. It's short for component set (http://www.cesm.ucar.edu/models/cesm2/config/compsets.html).

Comment (fairyeszhuo) [reply _episodes/08-simulations.md]: Get it. Maybe it's better to use the long name or maybe add (component set) here

**Comment by annefou** [comment]
Reverted changes for compset and accepted other changes. I will anyway update this episode for this year. Thanks.

**Pull request merged**